cephprimarystorage/cephprimarystorage/cephdriver.py

```
@@ -40,7 +40,10 @@ def create_volume(self, cmd, rsp, agent=None):
         if cmd.skipIfExisting and shell.run("rbd info %s" % path) == 0:
             return rsp
 
-        if ceph.is_xsky():
+        if cmd.systemTags and ceph.is_qcow2_format(cmd.systemTags):
+            vol_path = "rbd:%s" % path
+            linux.qcow2_create(vol_path, cmd.size, False)
```

Ruff

`linux` may be undefined, or defined from star imports

(F405)